hooks/post_gen_project.py

```diff
@@ -43,6 +43,55 @@ def delete_jenkins_ci():
     pass
 
 
+def readme_toc_update() -> None:
+    """Checks and updates toc in readme.md \n
```

Comment:
if you want to add a new line simply add one, instead of a `\n`

Reply:
I've added additional line and it shows description correctly